packages/lesswrong/lib/collections/doppelCommentVotes/schema.ts

```diff
@@ -0,0 +1,63 @@
+import { foreignKeyField, schemaDefaultValue } from "@/lib/utils/schemaUtils";
+import { userOwns } from "@/lib/vulcan-users";
+
+const schema: SchemaType<"DoppelCommentVotes"> = {
+  userId: {
+    ...foreignKeyField({
+      idFieldName: "userId",
+      collectionName: "Users",
+      resolverName: "user",
+      type: "User",
+      nullable: false,
```

Comment:
This should probably be `nullable: true`, because it controls type code gen, and for permissions-related reasons it's often the case that referenced documents are often inaccessible to users who have access to their ID via another type of document.  (In this case that seems unlikely, but it's just much safer to assume that resolved field values that need to go through any sort of permissions checks are nullable.)